packages/rspack-plugin-react-refresh/src/options.ts

```diff
@@ -9,6 +9,7 @@ const d = <K extends keyof PluginOptions>(
 	property: K,
 	defaultValue?: PluginOptions[K]
 ) => {
+	// TODO: should we also add default for null?
```

Comment:
Null is used to ignore the default value.

Reply:
Thanks! But I found the plugin config does not accept null. Maybe we should change the definition?

Reply:
Did you mean `PluginOptions['include']` and `PluginOptions['exclude']`? They all support passing `null`.

https://github.com/web-infra-dev/rspack/blob/dbb9927d1bb3dd972266dddc61dc26a1d0304d84/packages/rspack-plugin-react-refresh/src/options.ts#L2